src/main/java/org/broadinstitute/consent/http/mail/freemarker/NewDaaRequestModel.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.consent.http.mail.freemarker;
+
+import java.util.List;
+import java.util.Map;
+import org.broadinstitute.consent.http.models.Dac;
+import org.broadinstitute.consent.http.models.Dataset;
+
+public class NewDaaRequestModel {
+
+  private final String serverUrl;
+  private final String daaName;
+  private final String userName;
+  private final String signingOfficialUserName;
+  public NewDaaRequestModel(
+      String serverUrl,
+      String daaName,
+      String userName,
+      String signingOfficialUserName
+  ) {
+    this.serverUrl = serverUrl;
+    this.daaName = daaName;
+    this.userName = userName;
+    this.signingOfficialUserName = signingOfficialUserName;
+  }
+
+  public String getServerUrl() {
+    return serverUrl;
+  }
+
+  public String getDaaName() {
+    return daaName;
+  }
+
+  public String getUserName() {
+    return userName;
+  }
+
+  public String getSigningOfficialUserName() {
+    return signingOfficialUserName;
+  }
```

Comment:
[FYI]

I'm not sure if they're used in Consent, but this model looks like a good use case for definition via a Java record:

```suggestion
public record NewDaaRequestModel(String serverUrl, String daaName, String userName, String signingOfficialUserName) {}
```

Reply:
I'm getting FreeMarker errors when changing this definition to a Java record. I'll create a ticket to change all of our FreeMarkerTemplate models to be defined as Records separately.